Provider/ImageProvider.php

```diff
@@ -174,6 +174,10 @@ public function generatePublicUrl(MediaInterface $media, $format)
      */
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
+        if ($format == 'reference') {
```

Comment:
@NeuralClone, I asked for Yoda condition because it's a current Symfony's convention as I pointed in response to @Soullivaneuh comment.